frontend/tests/e2e_tests/integration/03-files.spec.ts

```diff
@@ -31,12 +31,26 @@ const fileLocation = `fixtures/${fileName}`;
 
 test.describe('Files', () => {
   let navbar;
+  test.beforeAll(async () => {
+    // download a fresh version of the demo artifact and upload in any case (even though)
```

Comment:
"Unfinished comment here? (I noticed it wasn’t completed before) :)

Reply:
👌 cleaned up now